src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.length > 1 && style.includes(':'));
+
+  return styles.reduce((acc, style) => {
+    const [rawKey, rawValue] = style.split(':');
+
+    acc[rawKey.trim()] = rawValue.trim();
+
+    return acc;
```

Comment:
The use of `reduce` here is good practice for transforming an array into an object. However, consider using more descriptive variable names instead of `acc` for the accumulator to improve code readability, such as `styleObject` or `cssObject`.